person/views.py

```diff
@@ -78,16 +78,31 @@ def get_context_data(self, **kwargs):
                 command_chain = {}
 
                 last_cited = repr(membership.object_ref.lastciteddate.get_value().value)
-                node_list_raw = get_org_hierarchy_by_id(org.uuid, when=last_cited)
-                edge_list = get_command_edges(org.uuid, when=last_cited)
+                node_list_raw = get_chart_nodes(org.uuid, when=last_cited)
+                edge_list = get_chart_edges(org.uuid, when=last_cited)
+
+                # Create a "from" link for the person and their org.
+                edge_list.append({'from': str(org.uuid)})
```

Comment:
Don't chart edges need values for both `from` and `to`? What happens if there's just `from`?

Reply:
It still works! Since its the bottom node, it does not need both values.

Reply:
Oooooooooh right! Makes so much sense!